docs/source/docs/calibration/calibration.rst

```diff
@@ -3,9 +3,9 @@ Calibrating Your Camera
 
 .. important:: In order to detect AprilTags and use 3D mode, your camera must be calibrated at the desired resolution! Inaccurate calibration will lead to poor performance.
 
-To calibrate a camera, images of a chessboard (or grid of dots, or other target) are taken. by comparing where the grid corners (or dots) should be in object space (for example, a dot once every inch in an 8x6 grid) with where they appear in the camera image, we can find a least-squares estimate for intrinsic camera properties like focal lengths, center point, and distortion coefficients. For more on camera calibration, please review the `OpenCV documentation <https://docs.opencv.org/4.x/dc/dbb/tutorial_py_calibration.html>`_.
+To calibrate a camera, images of a Charuco board (or chessboard, grid of dots, or other target) are taken. By comparing where the grid corners (or dots) should be in object space (for example, a dot once every inch in an 8x6 grid) with where they appear in the camera image, we can find a least-squares estimate for intrinsic camera properties like focal lengths, center point, and distortion coefficients. For more on camera calibration, please review the `OpenCV documentation <https://docs.opencv.org/4.x/dc/dbb/tutorial_py_calibration.html>`_.
 
-.. warning:: While any resolution can be calibrated, resolutions lower than 960x720 are often too low to provide accurate results. Additionally, high resolutions may be too performance intensive for a coprocessor like a Raspberry Pi to handle (solutions to this are being looked into). Thus, we recommend 960x720 when using 3D mode.
+.. warning:: While any resolution can be calibrated, resolutions lower than 960x720 are often too low to provide accurate results. Additionally, high resolutions may be too performance-intensive for a coprocessor like a Raspberry Pi to handle (solutions to this are being looked into). Thus, we recommend 960x720 when using 3D mode.
```

Comment:
Is this still accurate with orange pi's?

Reply:
I believe that with RPI 4, 5, and orange pi 5 something like 1280*720ish is a good resolution to recommend I would instead recommend we say that the higher the resolution the worse the performance.

Reply:
I believe that with RPI 4, 5, and orange pi 5 something like 1280*720ish is a good resolution to recommend I would instead recommend we say that the higher the resolution the worse the performance.

Reply:
Yeah, let's avoid naming numbers if we can. "Go forth and do science" or whatever